### ISSUE TITLE
Webrick handler tries to modify frozen strings with the latest webrick gem

### ISSUE THREAD

**Issue by Fjan** [rack/rack#2030]
The version of webrick that was released today now has frozen strings (https://github.com/ruby/webrick/commit/9279dd8c61d4fa1d76f1c448cd2c60bbd7413a6a). This causes an exception when using webrick with rack: 
```
ERROR FrozenError: can't modify frozen String: ""
/ruby-3.1.2/gems/rack-2.2.6.2/lib/rack/handler/webrick.rb:120:in `block in service'
```

(I think the offending code on that line is `res.body << part`, it can perhaps simply be replaced by `res.body += part` but I don't know enough about what's going on there)

**Comment by ioquatix** [comment]
It's fixed in <https://github.com/ruby/webrick/pull/103> and I hope to release it soon.

**Comment by ioquatix** [comment]
Okay, it was just released in webrick v1.8.1.

**Comment by Fjan** [comment]
Thanks @ioquatix !
